Project.toml

```diff
@@ -1,12 +1,14 @@
 name = "POMDPXFiles"
 uuid = "c6f6ee83-58c6-5336-a19f-2c76817e1af6"
 repo = "https://github.com/JuliaPOMDP/POMDPXFiles.jl"
-version = "0.2.3"
+version = "0.2.5"
 
 [deps]
 LightXML = "9c8b4983-aa76-5018-a973-4c85ecc9e179"
+POMDPModels = "355abbd5-f08e-5560-ac9e-8b5f2592a0ca"
```

Comment:
I think this is only used for tests. I think `POMDPModels` should probably be under `[extras]` and there should be a `[targets]` section, similar to what is done in https://github.com/JuliaTeX/PGFPlots.jl/blob/master/Project.toml
It is probably best not to include test-only dependencies in `[deps]`.